certified-connectors/Nextset/apiDefinition.swagger.json

```diff
@@ -0,0 +1,97 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0.0",
+    "title": "NextSet_WorkFlow",
+    "description": "We provide our customers with features that make in-house application and approval operations more convenient.",
+    "contact": {
+      "email": "contact-info@nextset.co.jp",
+      "name": "NextSet",
+      "url": "https://www.nextset.co.jp/contact/"
+    }
+  },
+  "host": "prod-21.japanwest.logic.azure.com",
+  "basePath": "/workflows/8feeccae04e94f3586e2ba0cb5aef4da/triggers",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/manual/paths/invoke": {
+      "post": {
+        "description": "ロジック アプリの実行をトリガーします。",
```

Comment:
@taiSateraito We do not allow any swagger content in Non-English language. Can you please update them in English language characters?